README.adoc

```diff
@@ -231,21 +245,21 @@ The [hotspot=36-44 file=1]`testPreflightCorsRequest` test simulates a client sen
 The request is an [hotspot=39 file=1]`OPTIONS` HTTP request with the following headers:
 
 |===
-| *Header* | *Value*
-| Origin | anywebsiteyoulike.com
-| Access-Control-Request-Method | [hotspot=20 file=0]`DELETE`
-| Access-Control-Request-Headers | [hotspot=22 file=0]`MyOwnHeader2`
+| *Request Header* | *Request Value*
+| Origin | The value is set to `anywebsiteyoulike.com`. Indicates that the request originates from `anywebsiteyoulike.com`.
+| Access-Control-Request-Method | The value is set to `DELETE`. Indicates that the HTTP DELETE method will be used in the actual request.
+| Access-Control-Request-Headers | The value is set to `MyOwnHeader2`. Indicates the header `MyOwnHeader2` will be used in the actual request.
 |===
 
 Expect the following response headers and values if the preflight CORS request is successful, and the server is correctly configured:
 
 |===
-| *Header* | *Value*
-| Access-Control-Allow-Max-Age | [hotspot=23 file=0]`10`
-| Access-Control-Allow-Origin | anywebsiteyoulike.com
-| Access-Control-Allow-Methods | [hotspot=20 file=0]`OPTIONS, DELETE`
-| Access-Control-Allow-Credentials | [hotspot=21 file=0]`true`
-| Access-Control-Allow-Headers | [hotspot=22 file=0]`MyOwnHeader1, MyOwnHeader2`
+| *Response Header* | *Response Value*
+| Access-Control-Allow-Max-Age | The expected value is `10`. Indicates that the preflight request can be cached within `10` seconds.
+| Access-Control-Allow-Origin | The expected value is `anywebsiteyoulike.com`. Indicates whether a resource can be shared based on the returning value of the Origin request header `anywebsiteyoulike.com`.
+| Access-Control-Allow-Methods | The expected value is `OPTIONS, DELETE`.  Indicates that HTTP OPTIONS and DELETE methods can be used in the actual request.
```

Comment:
There are two spaces after the period on this line.

Reply:
I don't see which line you mean

Reply:
Removed extra spacing on Line 260!